### ISSUE TITLE
Resilio Sync 2.6+ have moved directory_root  option from webui section to root

### ISSUE THREAD

**Issue by juskrey** [NixOS/nixpkgs#93460]
Build ends up with error
```"Error while parsing config file: Invalid key context: 'directory_root' must be in global config section"```

https://github.com/NixOS/nixpkgs/blob/bb8f0cc2279934cc2274afb6d0941de30b6187ae/nixos/modules/services/networking/resilio.nix#L38

**Comment by davidak** [comment]
That don't solves the problem, but have you considered using [Syncthing](https://syncthing.net/)? It's very similar, but open source.

**Comment by juskrey** [comment]
> That don't solves the problem, but have you considered using [Syncthing](https://syncthing.net/)? It's very similar, but open source.

Well yeah, still Resilio was a somewhat better fit for me, also has iOS app

**Comment by danieldk** [comment]
@juskrey thanks for raising this issue again! I made two PRs before that solved this issue (#62948 and #85689), but they never got any reviews. I guess there are not a lot of Resilio users among Nixians :/. I still prefer it's usability over SyncThing and I also need an iOS app. At any rate, I have merged #85689, so it should be fixed now.

Thanks again for your contribution! And for reminding me to get that old PR moving again.
